antipodal_gripper_controller/src/antipodal_gripper_action_controller_parameters.yaml

```diff
@@ -0,0 +1,53 @@
+antipodal_gripper_action_controller:
+  action_monitor_rate: {
+    type: double,
+    default_value: 20.0,
+    description: "Hz",
+    validation: {
+      gt_eq: [ 0.1 ]
+    },
+  }
+  joint: {
+    type: string,
+    default_value: "",
+  }
+  goal_tolerance: {
+    type: double,
+    default_value: 0.01,
+    validation: {
+      gt_eq: [ 0.0 ]
+    },
+  }
+  max_effort: {
+    type: double,
+    default_value: 0.0,
+    description: "Max allowable effort",
+    validation: {
+      gt_eq: [ 0.0 ]
+    },
+  }
```

Comment:
My main goal by renaming these is to not confuse users between these and the actual `velocity`/`effort` state/command interfaces

Reply:
I added the target max_velocity` parameter, good catch!

Regarding your second comment, that is why I added the gripper prefix. The interfaces would show up with ros2_control add
```
command_interfaces:
  robotiq_85_left_knuckle_joint/gripper_speed
  robotiq_85_left_knuckle_joint/gripper_effort
```
As opposed to standard interfaces:  
```
command_interfaces:
  robotiq_85_left_knuckle_joint/velocity
  robotiq_85_left_knuckle_joint/effort
```
Because they are command interfaces, it seems prefixing with set_gripper could be redundant.

Reply:
Okay, I change the interface names to `set_gripper_max_velocity` and `set_gripper_max_effort`